app/pastes/create-paste/createPasteInteractor.js

```diff
@@ -23,10 +25,15 @@ class CreatePasteInteractor {
             text: request.text,
             visibility: request.visibility,
             expiresAfter: request.expiresAfter,
-            authorId: request.userId
+            authorId: request.userId,
+            totalViews: 0
         });
         await this.pasteRepository.save(paste);
 
+        const user = await this.userRepository.findByID({ id: request.userId });
```

Comment:
Actually, we should check if user even exists before setting him as paste's author